public/pages/CreateRollup/components/ScheduleRolesAndNotifications/ScheduleRolesAndNotifications.tsx

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import React, { Component } from "react";
+import { EuiFlexGrid, EuiFlexItem, EuiSpacer, EuiText } from "@elastic/eui";
+import { ContentPanel, ContentPanelActions } from "../../../../components/ContentPanel";
+import { ModalConsumer } from "../../../../components/Modal";
+import { parseTimeunit } from "../../utils/helpers";
+
+interface ScheduleRolesAndNotificationsProps {
+  rollupId: string;
+  onChangeStep: (step: number) => void;
+  jobEnabledByDefault: boolean;
+  recurringJob: string;
+  recurringDefinition: string;
```

Comment:
Let's change all occurrences of the word `recurring` to `continuous` to match up with the data models and definition of the continuous rollups.